x-pack/metricbeat/module/airflow/statsd/data_test.go

```diff
@@ -70,15 +70,18 @@ func TestData(t *testing.T) {
 		wg.Done()
 
 		go ms.Run(reporter)
-		events = reporter.(*mbtest.CapturingPushReporterV2).BlockingCapture(1)
+		events = reporter.(*mbtest.CapturingPushReporterV2).BlockingCapture(2)
 
 		close(done)
 	}(wg)
 
 	wg.Wait()
-	createEvent(t)
+	createEvent("dagrun.duration.failed.a_dagid:200|ms|#k1:v1,k2:v2", t)
+	createEvent("dagrun.duration.failed.b_dagid:500|ms|#k3:v3,k4:v4", t)
 	<-done
 
+	assert.Len(t, events, 2)
```

Comment:
does this test fail without the new changes in this PR?

Reply:
Missed running the test before making this change, but I've run it now and made the necessary updates(the tags needed to be identical for both the events to test this change). Thanks!